assets/js/common_utils.js

```diff
@@ -381,6 +387,10 @@ export function convertToFloat32Array(fp16_array) {
 
 // convert Float32Array to Uint16Array
 export function convertToUint16Array(fp32_array) {
+    if (isFloat16ArrayAvailable) {
+        // Float16Array is available - use it
+        return Float16Array.from(fp32_array, v => v);
```

Comment:
`convertToUint16Array` - We should reevaluate the name now, because it doesn't reliably return a Uint16Array anymore.

Reply:
Done, PTAL again, thanks!